### PR TITLE
[15.0][MIG]product_analytic_purchase: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by bizzappdev** [comment]
ready for review

File: product_analytic_purchase/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Product Analytic Purchase",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (rousseldenis) [inline product_analytic_purchase/__manifest__.py:6]: @bizzappdev Could you reset to 15.0.1.0.0 ?

**Review by rousseldenis — APPROVED** [review]
Minor comment

**Pull request closed without merging**